extensions/database/tests/src/com/google/refine/extension/database/DBExtensionTests.java

```diff
@@ -43,7 +43,7 @@
     protected final String DEFAULT_MYSQL_HOST = "127.0.0.1";
     protected final String DEFAULT_MYSQL_PORT = "3306";
     protected final String DEFAULT_MYSQL_USER = "root";
-    protected final String DEFAULT_MYSQL_PASSWORD = "secret";
+    protected final String DEFAULT_MYSQL_PASSWORD = "";
```

Comment:
We should probably keep this and use the XML files to change the password when needed.